server/controllers/medical/reports/visits.report.handlebars

```diff
@@ -0,0 +1,77 @@
+{{> head title="PATIENT_RECORDS.REPORT.VISITS"}}
+
+<style>
+  th.rotate {
+    height: 140px;
+    white-space: nowrap;
+    padding-bottom: 10px !important;
+  }
+  th.rotate > div {
+    -webkit-transform: rotate(-90deg);
+            transform: rotate(-90deg);
+    width: 30px;
+  }
+</style>
+
+<div class="container">
+  {{> enterpriseDetails }}
+
+  <div class="row">
+    <div class="col-xs-12">
+
+      <h3 class="text-center">{{translate 'PATIENT_RECORDS.REPORT.VISITS'}}</h3>
+      <h4 class="text-center">
+        <small>{{date details.dateFrom }}</small> - <small>{{date details.dateTo }}</small>
+      </h4>
+
+      <table class="table table-report">
+        <thead>
+          <tr class="text-capitalize">
+            <th class="text-center" style="width: 35%;">{{translate "FORM.LABELS.SERVICE"}}</th>
+            <th class="rotate"><div><span>{{translate "FORM.LABELS.TOTAL"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.NEW_CASES"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.OLD_CASES"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.HOSPITALIZED"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.AMBULATORY"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.PREGNANT"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.REFERRED"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.INSIDE_HEALTH_ZONE"}}</span></div></th>
+            <th class="rotate"><div><span>{{translate "PATIENT_RECORDS.REPORT.OUTSIDE_HEALTH_ZONE"}}</span></div></th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each visits as |visit|}}
+            <tr>
+              <td>{{ visit.service_name }}</td>
+              <td>{{ visit.total }}</td>
+              <td>{{ visit.total_new_case }}</td>
+              <td>{{ visit.total_old_case }}</td>
+              <td>{{ visit.total_hospitalized }}</td>
+              <td>{{ visit.total_ambulatory }}</td>
+              <td>{{ visit.total_pregnant }}</td>
+              <td>{{ visit.total_referred }}</td>
+              <td>{{ visit.total_inside_health_zone }}</td>
+              <td>{{ visit.total_outside_health_zone }}</td>
+            </tr>
+          {{else}}
+            {{> emptyTable columns=6}}
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr>
+            <th></th>
+            <th>{{sum visits 'total'}}</th>
```

Comment:
I'm always a fan of using SQL instead of JS for summing.  If you rewrite your query in the way I suggest below, it should be very easy to use SQL to calculate these values by just removing the GROUP BY service.